lib/src/emoji_lists.dart

```diff
@@ -2602,17 +2602,17 @@ final Map<String, String> symbols = Map.fromIterables([
   '©',
   '®',
   '™',
-  '#️⃣',
-  '0️⃣',
-  '1️⃣',
-  '2️⃣',
-  '3️⃣',
-  '4️⃣',
-  '5️⃣',
-  '6️⃣',
-  '7️⃣',
-  '8️⃣',
-  '9️⃣',
+  '#⃣',
+  '0⃣',
+  '1⃣',
+  '2⃣',
+  '3⃣',
+  '4⃣',
+  '5⃣',
+  '6⃣',
+  '7⃣',
+  '8⃣',
+  '9⃣',
```

Comment:
Can you explain me the improvement here? It became non-displayable in MacOS and Android didn't change

Reply:
That is strange. I only tested with Android, but the reason why I wanted to make the change was because I wanted to use the emoji picker together with a Twemoji library (Twitter emojis) and this was the only way to make the number emojis display. Though it might be a bug in the Twemoji library as I had to make a pull request there too with some Unicode fixes. I will look into it as they are most likely broken with the changes I made.

Reply:
It looks like Twemoji leaves the middle codepoint out for some reason for 0-9 and #. When I tested the changes I did on Android the Native Material emojis automatically got resolved as "U+23 U+FE0F U+20E3" instead of "U+23 U+FE0F U+20E3" which made it look like it was working for both platforms. So in summary **discard** the changes I made here.

Reply:
I'm not really understanding all of this 😅, but it's your PR and when this change can be reverted would be great if you could do it. 🙏

Reply:
Sorry my bad! I have reverted it now. 😄